Task/Workflow/TSKWorkflow.h

```diff
@@ -219,6 +225,39 @@ extern NSString *const TSKWorkflowTaskKey;
 - (void)addTask:(TSKTask *)task prerequisiteTasks:(NSSet *)prerequisiteTasks;
 
 /*!
+ @abstract Adds the specified task to the task workflow with the specified dictionary of keyed 
+     prerequisite tasks.
+ @discussion The task’s workflow property is set to the receiver, and its keyed prerequisite tasks are
+     set to the ones specified. Furthermore, the task is added to each of the keyed prerequisite tasks’ 
+     sets of dependent tasks. If the task has any prerequisites, its state is set to pending.
+
+     This is not a thread-safe operation. This method should only execute on one thread at a time.
+ @param task The task to add. May not be nil. May not be a member of any other task workflow.
+ @param keyedPrerequisiteTasks A dictionary that maps the task’s prerequisite names to their 
+     corresponding task. If nil, the task will have no keyed prerequisite tasks. Otherwise, each task
+     in the dictionary must have already been added to the receiver.
+ */
```

Comment:
Will likely update the discussion to explain that it’s equivalent to invoking `addTask:prerequisiteTasks:keyedPrerequisiteTasks:`.